blog/2023-06-15-tauri-board-elections-and-governance-updates.mdx

```diff
@@ -0,0 +1,61 @@
+
+Voting will take place starting on July 5th through July 12th, 2023 where Tauri Working Group Members will cast their votes. We're expecting to announce the results on or before July 16, 2023. If elected, we'll ask you to sign [this pledge](https://dracc.commonsconservancy.org/0016/) if you haven't done so previously.
+
+If you have any questions or would like more information please reach out by emailing [board@tauri.app](mailto:board@tauri.app), messaging the `@board` role on the [Tauri Discord](https://discord.com/invite/tauri), or by directly messaging Daniel Thompson-Yvetot on Discord (`@Denjell`) or email ([denjell@tauri.app](mailto:denjell@tauri.app)).
+
+---
+
+## Governance Update
+
+Tauri has always had a Working Group that's responsible for the day-to-day direction and execution, but as we've grew it became clear that we needed to evolve the Working Group. We also wanted to give new Working Group members a clear places to jump in and get started. We thought of how we could achieve this while building and iterating on the existing framework.  **With these goals in mind, we created two new concepts within the Working Group: Domains and Domain Leads.**
+
+### Domains
+
+A Domain in the Working Group represents a specific area of interest. We're starting with these 4 domains:
+
+- **Development**: Developing and maintaining the Tauri software
+- **Community**: Looks after the broader community resources and the public presence of Tauri
+- **Governance & Guidance**: Involved with all things organizational in nature
+- **Operations**: Responsible for all the tooling and infrastructure Tauri needs to get work done
+
+This gives clear entry points for a new member of the Working Group to jump into right away. It also allows us to continue growing our core competency of core Tauri development while giving additional focus to all areas which support the Tauri organization as a whole. Each of these domains will be represented by Domain Members and Domain Leads, but membership is not mutually exclusive.
+
+### Domain Leads
+
+Within a domain, there is a position for 2-3 Domain Leads. These are core individuals in the Working Group that are critical to their respective Domain. They help by setting the direction and priorities for their domain while also serving as key leaders for the Working Group as a whole.
+
+Leads are elected twice a year, in the spring and in the fall. Since this governance update is being rolled out mid-term we're appointing interim leads that have been key members of the Working Group. This fall we'll hold the first round of elections to formally elect the Domain Leads. We'll announce information for that process when we're closer to election season.
```

Comment:
```suggestion
Leads are appointed twice a year, in the spring and in the fall. Since this governance update is being rolled out mid-term we're appointing interim leads that have been key members of the Working Group. This fall we'll reflect on the initial experience and host a new round to appoint the Domain Leads. We'll announce information for that process when we're closer to "appointment season".
```

Important detail this. The Board appoints Domain Leads. So calling that *elected* could suggest that it's the WG doing the electing. Appointment is also the term that is used in regulation, which is leading in this regard.

Also slight rewording to try and express what that round would look like in fall. Primarily a reflection moment, a call for new applicants and another "term" for appointments.

(They're not formalized terms, but soft terms)